docs/reference/announcements-release-notes/880/880-announcements.md

```diff
@@ -1,65 +1,97 @@
 ---
-id: announcements-880
-title: "8.8 Announcements"
-description: "Important changes and updates for the Camunda 8.8 release including deprecation & removal notices."
+id: 880-announcements
+title: "Announcements"
+description: "Supported environment changes and breaking changes or deprecations for the Camunda 8.8 release."
+toc_max_heading_level: 2
 ---
 
-import DeployDiagramImg from '../img/deploy-diagram-modal.png';
+import DeployDiagramImg from '../../img/deploy-diagram-modal.png';
+import AnnouncementsGrid from '../../react-components/\_announcements-card.js';
+import { AnnouncementsCards } from '../../react-components/\_announcements-card-data';
 
-Important changes and updates for the Camunda 8.8 release are summarized below.
+Supported environment changes and breaking changes or deprecations for the Camunda 8.8 release are summarized below.
+
+[Release notes](/reference/announcements-release-notes/880/880-release-notes.md) now contain all updates for new features and enhancements. Click any of the features below to be re-routed to the release notes.
+
+<AnnouncementsGrid announcements={AnnouncementsCards} />
 
 | Scheduled release date | Scheduled end of maintenance | Release notes | Blog |
 | :--------------------- | :--------------------------- | :------------ | :--- |
 | 14 October 2025        | 13 April 2027                | -             | -    |
 
-- [API updates](#api-updates-saasself-managed)
-- [Identity management updates](#identity-management-updates-saasself-managed)
-- [Installation and deployment updates](#installation-and-deployment-updates-self-managed)
-- [Camunda Exporter](#camunda-exporter-self-managed)
-- [Camunda Java client and Camunda Spring SDK](#camunda-java-client-and-camunda-spring-sdk-self-managed)
+- [Supported environment changes](#supported-environment-changes)
+  - [Zeebe, Operate, Tasklist, and Identity must run on exact same minor and patch levels](#zeebe-operate-tasklist-and-identity-must-run-on-exact-same-minor-and-patch-levels)
+  - [Installation and deployment updates Self-Managed](#installation-and-deployment-updates-self-managed)
+    - [Helm charts](#helm-charts)
+- [Breaking changes](#breaking-changes)
+  - [API updates SaaSSelf-Managed](#api-updates-saasself-managed)
+    - [Deprecated: Operate and Tasklist v1 REST APIs](#deprecated-operate-and-tasklist-v1-rest-apis)
+    - [Deprecated: Job-based user tasks querying](#deprecated-job-based-user-tasks-querying)
+    - [Deprecated: Zeebe gRPC API endpoints](#deprecated-zeebe-grpc-api-endpoints)
+    - [Removed: Tasklist GraphQL API](#removed-tasklist-graphql-api)
+    - [Removed: Deprecated OpenAPI objects](#removed-deprecated-openapi-objects)
+  - [Camunda Java client and Camunda Spring SDK](#camunda-java-client-and-camunda-spring-sdk)
+    - [Key changes](#key-changes)
+
+## Supported environment changes
+
+### Zeebe, Operate, Tasklist, and Identity must run on exact same minor and patch levels
```

Comment:
One cannot see the ### (h3) titles in the navigation on the right hand side. Can we adjust this? 
(Or alternatively: change `## Supported environment changes` to `# Supported environment changes`

Reply:
Adjusting in incoming commit 👍